staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -580,6 +580,9 @@ func validateCustomResourceConversion(conversion *apiextensions.CustomResourceCo
 			case cc.Service != nil:
 				allErrs = append(allErrs, webhook.ValidateWebhookService(fldPath.Child("webhookClientConfig").Child("service"), cc.Service.Name, cc.Service.Namespace, cc.Service.Path, cc.Service.Port)...)
 			}
+			if len(cc.CABundle) > 0 {
```

Comment:
we want to make this ratcheting, so plumb down a `CustomResourceConversionValidationOptions { AllowInvalidCABundle bool }` field, set to false on create and set to true on update *only* if the existing CABundle is invalid

Reply:
Updated, thanks for the pointer.